packages/app-store/googlecalendar/lib/CalendarService.ts

```diff
@@ -194,9 +195,7 @@ export default class GoogleCalendarService implements Calendar {
       calendar.events.update(
         {
           auth: myGoogleAuth,
-          calendarId: event.destinationCalendar?.externalId
-            ? event.destinationCalendar.externalId
-            : "primary",
+          calendarId: externalCalendarId ? externalCalendarId : event.destinationCalendar?.externalId,
```

Comment:
If an extneral calendar id is not pased then use the current destination calendar id